app/client/src/constants.js

```diff
@@ -1,6 +1,6 @@
 angular.module('reg')
     .constant('EVENT_INFO', {
-        NAME: 'HackUCI 2019',
+        NAME: 'HackUCI 2010',
         START_DATE: 'Feb 15, 2019',
         START_TIME: '22:00',
         END_DATE: 'Feb 17, 2019',
```

Comment:
2020?
and for start and end time might as well change it now
Jan. 31st-Feb 2nd!